Source/SA/RFmx SpecAn/RFmxSpecAn.cs

```diff
@@ -44,33 +40,313 @@ public struct AutoLevelConfiguration
         {

```

Comment:
why did we separate AutoLevelConfiguration from the CommonConfiguration? The other modules (NR) keep them together.

Reply:
That was probably my fault. Some of the old code used to do that (WLAN). I just removed it.